fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/query/plan/debug/PlannerRepl.java

```diff
@@ -80,7 +80,9 @@ public class PlannerRepl implements Debugger {
             "88.  .88 88 88.  .88 88    88 88    88 88.  ... 88      \n" +
             "88Y888P' dP `88888P8 dP    dP dP    dP `88888P' dP      \n" +
             "88                                                      \n" +
-            "dP                                                      \n";
+            "dP                                                      \n" +
```

Comment:
Want to give the whole thing a new banner? `planner` is so bland :-)

Reply:
I kinda like It honestly 😄 , but please let me know if you have something better, something like this?

```
╭━━━╮╱╱╱╱╱╱╱╱╱╱╱╱╱╭╮╱╱╱╱╱╱╭━━━╮╱╱╭╮
┃╭━╮┃╱╱╱╱╱╱╱╱╱╱╱╱╱┃┃╱╱╱╱╱╱╰╮╭╮┃╱╱┃┃
┃┃╱╰╋━━┳━━┳━━┳━━┳━╯┣━━┳━━╮╱┃┃┃┣━━┫╰━┳╮╭┳━━┳━━┳━━┳━╮
┃┃╱╭┫╭╮┃━━┫╭━┫╭╮┃╭╮┃┃━┫━━┫╱┃┃┃┃┃━┫╭╮┃┃┃┃╭╮┃╭╮┃┃━┫╭╯
┃╰━╯┃╭╮┣━━┃╰━┫╭╮┃╰╯┃┃━╋━━┃╭╯╰╯┃┃━┫╰╯┃╰╯┃╰╯┃╰╯┃┃━┫┃
╰━━━┻╯╰┻━━┻━━┻╯╰┻━━┻━━┻━━╯╰━━━┻━━┻━━┻━━┻━╮┣━╮┣━━┻╯
╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╱╭━╯┣━╯┃
```

Reply:
party time -- excellent

Reply:
not for now -- it's ok.